content/en/docs/reference/using-api/cel.md

```diff
@@ -338,30 +746,75 @@ Only Kubernetes resource property names of the form
 names are escaped according to the following rules when accessed in the
 expression:
 
-{{< table caption="Table of CEL identifier escaping rules" >}}
-| escape sequence   | property name equivalent                                                                     |
-|-------------------|----------------------------------------------------------------------------------------------|
-| `__underscores__` | `__`                                                                                         |
-| `__dot__`         | `.`                                                                                          |
-| `__dash__`        | `-`                                                                                          |
-| `__slash__`       | `/`                                                                                          |
-| `__{keyword}__`   | [CEL **RESERVED** keyword](https://github.com/google/cel-spec/blob/v0.6.0/doc/langdef.md#syntax) |
-{{< /table >}}
+<table>
+<caption>Table of CEL identifier escaping rules</caption>
+<thead>
+<tr>
+  <th>escape sequence</th>
+  <th>property name equivalent</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td><tt>\_\_underscores\_\_</tt></td>
+  <td><tt>\_\_</tt></td>
```

Comment:
```suggestion
  <td><tt>__underscores__</tt></td>
  <td><tt>__</tt></td>
```
It seems these escape characters `\` should be removed from the HTML, as the current display is incorrect.